openml/openml-python
# added serialize run functionality

#### What does this PR implement/fix? Explain your changes.
It allows run objects (including predictions and traces) to be serialized to disk, and reloaded. This functionality is almost used in all my projects, and I can imagine us using this for the benchmark study. 

#### How should this PR be tested?
Unit tests should pass, code should make sense, please check the unit test if run equality check is OK. 

#### Any other comments?
